include/alpaka/vec/Vec.hpp

```diff
@@ -988,7 +1014,9 @@ namespace alpaka
             using IdxOffset = std::integral_constant<std::intmax_t, static_cast<std::intmax_t>(dim::Dim<TExtent>::value) - static_cast<std::intmax_t>(TDim::value)>;
             return
 #ifdef ALPAKA_CREATE_VEC_IN_CLASS
-            Vec<TDim, size::Size<TExtent>>::template
+                Vec<TDim, size::Size<TExtent>>::template
```

Comment:
We should do this change now even if it need a change in the user code. This is only a change in the current dev which goes to the next release. We should add a no with interface changes to the release notes of the next release.